test/ripple-client-test.ts

```diff
@@ -27,11 +27,30 @@ describe('Client [Test Runner]', function () {
 
   // Collect all the tests:
   const allPublicMethods = getAllPublicMethods(new Client())
-  const allTestSuites = loadTestSuites()
+  const utilTestSuites = loadTestSuitesFromFolder("utils")
+  const apiTestSuites = loadTestSuitesFromFolder("client")
```

Comment:
`utils` tests don't need to be run in this format, they don't have to be run with two types of addresses and with the client.

Reply:
One test seems to require client.getLedger and I couldn't find any alternatives. Do you happen to know any ways to get a ledger without the Client object? (The specific test is `test/client/computeLedgerHash/index.ts` - I moved it back into the api tests so that it could have access to client until I learn a viable alternative).

Moved the rest into the default format though!